City-of-Helsinki/drupal-hdbt
# UHF-5287: UHF-5287 Move the calendar icon to used icons

# Move calendar icon to used icons [UHF-5287](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5287)
Calendar icon is required in the UHF-5287 task so moving it to used icons folder.

## What was done
* Calendar icon was moved to used icons folder.

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the HDBT theme
    * `composer require drupal/hdbt:dev-UHF-5287_calendar_icon`
* Run `make drush-cr`

## How to test
* Check this as part of the https://github.com/City-of-Helsinki/drupal-helfi-rekry/pull/19 review so on the job listing page you should see the calendar icon correctly before the title "Employment contract".

## Designers review

One of the checkboxes below needs to be checked (`[x]`)


## Other PRs
* https://github.com/City-of-Helsinki/drupal-helfi-rekry/pull/19
